desktop/core/src/desktop/js/reactComponents/FileChooserComponent/FileChooserComponentButton/FileChooserComponentButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './FileChooserComponentButton.scss';
+import FileChooserModal from '../FileChooserModal/FileChooserModal';
+import { Button } from 'antd';
+
+const FileChooserComponentButton = (): JSX.Element => {
+  const [show, setShow] = useState(false);
```

Comment:
As we move forward with the integration we probably want to lift the state into the page/component that actually uses the new FileChoser and we will be needing callbacks to handle the actions triggered by this component, but for now we can leave it.